kbs/src/api/src/http/config.rs

```diff
@@ -25,9 +25,6 @@ pub(crate) async fn attestation_policy(
     }
 
     attestation_service
-        .0
-        .lock()
-        .await
```

Comment:
Any concerns about not locking here given that this will change the state of the AS?

Reply:
Thanks for pointing out this. IMO, the lock/synchronization mechanism should be token cared of on the server side, like in CoCo-AS we use [a read-write lock](https://github.com/confidential-containers/kbs/blob/1cdd24d/attestation-service/bin/grpc-as/src/server.rs#L227) to handle the requests.

The client side code just call the API and the server can response in some order. That means, if users want to use a new policy to do attestation. In their code should call `set_policy` of KBS and wait for the successful return. Then call attestation API.

Among current three implementations of AS client, `built-in-coco-as` seems to have some trouble for synchronization. This is because the policy management is now based on the filesystem and we do not use a explicit lock/mutex which would make trouble. Let me add a Mutex on the client side as a workaround. In future we can fix this in the `attestation-service` further.